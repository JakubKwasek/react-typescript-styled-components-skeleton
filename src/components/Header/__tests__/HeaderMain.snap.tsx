import * as React from "react";
import * as renderer from "react-test-renderer";
import { BrowserRouter } from "react-router-dom";
import { HeaderMain } from "../";

describe("HeaderMain", () => {
	it("renders correctly", () => {
		const tree = renderer
			.create(<BrowserRouter><HeaderMain /></BrowserRouter>)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
