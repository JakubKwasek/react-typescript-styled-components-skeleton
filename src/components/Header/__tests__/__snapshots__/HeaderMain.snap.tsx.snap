// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderMain renders correctly 1`] = `
<header
  className="StyledBox-sc-13pk1d4-0 jgHovw"
>
  Movie Shop
  <div
    className="StyledBox__StyledBoxGap-sc-13pk1d4-1 bmrRmF"
  />
  <div
    className="StyledBox-sc-13pk1d4-0 eitMOG"
  >
    <a
      aria-current="page"
      className="is-active"
      href="/"
      onClick={[Function]}
      style={Object {}}
    >
      Dashboard
    </a>
    <div
      className="StyledBox__StyledBoxGap-sc-13pk1d4-1 bmrRmF"
    />
    <a
      aria-current={null}
      href="/store"
      onClick={[Function]}
    >
      Store
    </a>
    <div
      className="StyledBox__StyledBoxGap-sc-13pk1d4-1 bmrRmF"
    />
    <a
      aria-current={null}
      href="/cart"
      onClick={[Function]}
    >
      Cart
    </a>
  </div>
</header>
`;
